coalib/parsing/DefaultArgParser.py

```diff
@@ -170,6 +170,10 @@ def default_arg_parser(formatter_class=None):
             help="filters `--show-bears` by the given languages")
 
         outputs_group.add_argument(
+           '-x', '--show-capabilites', nargs='+', metavar='LANG',
+           help="show languages' bears capabilities")
```

Comment:
I don't fully get what this does, could you explain? I feel like the description needs to be  improved.